core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1234,17 +1235,24 @@ private void updateMediaSession(final PlayerStatus playerStatus) {
                         CUSTOM_ACTION_FAST_FORWARD,
                         getString(R.string.fast_forward_label), R.drawable.ic_notification_fast_forward)
                         .build());
+        } else {
+            // This would give the PIP of videos a play button
+            capabilities = capabilities | PlaybackStateCompat.ACTION_PLAY;
+            if (uiModeManager.getCurrentModeType() == Configuration.UI_MODE_TYPE_WATCH) {
+                flavorHelper.sessionStateAddActionForWear(sessionState,
+                        CUSTOM_ACTION_REWIND,
+                        getString(R.string.rewind_label),
+                        android.R.drawable.ic_media_rew);
+                flavorHelper.sessionStateAddActionForWear(sessionState,
+                        CUSTOM_ACTION_FAST_FORWARD,
+                        getString(R.string.fast_forward_label),
+                        android.R.drawable.ic_media_ff);
+                flavorHelper.mediaSessionSetExtraForWear(mediaSession);
+            }
         }
 
         sessionState.setActions(capabilities);
 
-        flavorHelper.sessionStateAddActionForWear(sessionState,
```

Comment:
Putting this into the else block, I saw duplicate rewind and fast forward on Autos